components/math/longestStreak/userTableStats.tsx

```diff
@@ -0,0 +1,129 @@
+import { useQuery } from "@apollo/client";
+import React, { useEffect } from "react";
+import { FetchGameLevelResponse } from "../../../graphql/longestStreak/fetchGameLevel";
+import { FETCH_RANKED_GAME_LEVEL } from "../../../graphql/longestStreak/fetchRankedGameLevel";
+import { Button } from "../../ui/Button";
+
+export interface UserTableStatsProps {
+  onClick: () => void;
+}
+
+export const UserTableStats: React.FC<UserTableStatsProps> = ({ onClick }) => {
+  const { data } = useQuery<FetchGameLevelResponse>(FETCH_RANKED_GAME_LEVEL, {
+    onCompleted: (data: FetchGameLevelResponse) => {
+      console.log(data);
+    },
+  });
+
+  return (
+    <>
+      <h1 className="flex justify-center md:bg-transparent md:p-none p-2 bg-purple-300">
+        Leader Dashboard
+      </h1>
+      <table
+        className="w-full text-left border border-separate rounded border-slate-200"
+        cellSpacing="0"
+      >
+        <tbody>
+          <tr>
+            <th
+              scope="col"
+              className="hidden h-12 px-6 text-sm font-medium bg-purple-300 border-l sm:table-cell first:border-l-0 stroke-slate-700 text-slate-700"
+            >
+              Name
+            </th>
+            <th
+              scope="col"
+              className="hidden h-12 px-6 text-sm font-medium bg-purple-300 border-l sm:table-cell first:border-l-0 stroke-slate-700 text-slate-700"
+            >
+              Games Won
+            </th>
+            <th
+              scope="col"
+              className="hidden h-12 px-6 text-sm font-medium bg-purple-300 border-l sm:table-cell first:border-l-0 stroke-slate-700 text-slate-700"
+            >
+              Current Level
+            </th>
+            <th
+              scope="col"
+              className="hidden h-12 px-6 text-sm font-medium bg-purple-300 border-l sm:table-cell first:border-l-0 stroke-slate-700 text-slate-700"
+            >
+              Games Played
+            </th>
+            <th
+              scope="col"
+              className="hidden h-12 px-6 text-sm font-medium bg-purple-300 border-l sm:table-cell first:border-l-0 stroke-slate-700 text-slate-700"
+            >
+              Games Lost
+            </th>
+          </tr>
+          {data &&
+            data.longestStreakUserData.map((row) => {
```

Comment:
Ok this table looks good, but did you need the other one

Reply:
Correct.  This is the working file.